### PR TITLE
Update config.js

### PR DESCRIPTION
**Pull request by Freddie-m**
This notification is missing in the array. If array is selectively defined, no pop-up notification is received when participant requests to join the meeting.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1205,6 +1205,7 @@ var config = {
     //     'liveStreaming.unavailableTitle', // shown when livestreaming service is not reachable
     //     'lobby.joinRejectedMessage', // shown when while in a lobby, user's request to join is rejected
     //     'lobby.notificationTitle', // shown when lobby is toggled and when join requests are allowed / denied
+    //     'notify.participantWantsToJoin', // shown when lobby is enabled and participant requests to join meeting
```
Comment (saghul) [inline config.js:1208]: Can you sort it alphabetically please?

Comment (Freddie-m) [reply config.js]: Sure. Done.

**Review by Freddie-m — COMMENTED** [review]
Sorted alphabetically

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Cheers!

**Pull request merged**